build.gradle

```diff
@@ -10,12 +10,14 @@ repositories {
 
 dependencies {
     compile(
-        "com.sparkjava:spark-core:2.9.3"
+        "com.sparkjava:spark-core:2.9.3",
+        "com.sparkjava:spark-template-mustache:2.7.1",
+        "com.nimbusds:oauth2-oidc-sdk:9.3.1"
     )
```

Comment:
You can enable logging by adding the `"org.slf4j:slf4j-simple:1.7.2"` dependency